src/views/Apply.vue

```diff
@@ -14,8 +14,9 @@
         />
       </div>
       <div class="form-item">
-        <p>Year:</p>
-        <select v-model="year">
+        <label for="year">Year</label>
+        <br />
```

Comment:
Long term, we want to use margin/padding instead of `<br />` tags.

Reply:
Ok, we should probably do this as a separate issue though because it affects the whole website

Reply:
certainly, just leaving as a note